src/lightmap.cpp

```diff
@@ -331,6 +331,15 @@ void map::generate_lightmap()
                 }
             }
         }
+        // atomic light is always on
+        std::vector<int> light_indices = v->all_parts_with_feature(VPFLAG_ATOMIC_LIGHT);
```

Comment:
If the vehicle actually has the part. I meant a flag to indicate the presence of the part -- calling `v->all_parts_with_feature(VPFLAG_ATOMIC_LIGHT)` every frame could potentially be expensive with a lot of vehicles in the area. Profiling said scenario would indicate if it's actually a problem or not, but a flag ought to be a simple change.